src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.js

```diff
@@ -48,51 +48,56 @@ function generator({ artifactTarball, versionTag, license, usePublicArtifact  })
   # REF: https://docs.openshift.org/latest/creating_images/guidelines.html
   RUN chmod -R g=u /usr/share/kibana
   RUN find /usr/share/kibana -type d -exec chmod g+s {} \\;
-  
+
   ################################################################################
   # Build stage 1
   # Copy prepared files from the previous stage and complete the image.
   ################################################################################
   FROM centos:7
   EXPOSE 5601
-  
+
   # Add Reporting dependencies.
-  RUN yum update -y && yum install -y fontconfig freetype && yum clean all
-  
+  RUN yum update -y && yum install -y fontconfig freetype wget && yum clean all
+
+  # Add an init process
+  RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && chmod +x /usr/local/bin/dumb-init
```

Comment:
Small thing, if you use `curl` here, you won't need to add `wget` to the image. Wouldn't call it a dealbreaker.

`curl -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64` should do the same thing